web/controllers/friends_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Entice.Web.FriendsController do
+  use Entice.Web.Web, :controller
+  alias Entice.Web.Queries
+  alias Entice.Web.Friend 
+
+
+
+  @doc "Returns all friends of connected account."
+  def index(conn, _params) do
+    id = get_session(conn, :client_id)
+
+    friendlist = Queries.get_friendlist(id)
```

Comment:
Afaics this function does not yet exist... maybe it's supposed to be an Entice.Web.Client function? (That would still need to be implemented ;))